third_party/move/move-core/types/src/transaction_argument.rs

```diff
@@ -19,6 +19,7 @@ pub enum TransactionArgument {
     U16(u16),
     U32(u32),
     U256(u256::U256),
+    Raw(Vec<u8>),
```

Comment:
What is the difference from `TransactionArgument::U8Vector`?

Reply:
TransactionArgument::U8Vector means that the value is of type `vector<u8>` in Move, whereas `Raw` is serialized move value and VM will check if this can be passed into the txn.